<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dishes</class>
 <widget class="QWidget" name="dishes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {
            height: 40px;
            width: 120px;
            font-size: 16px;
            padding: 10px;
        }
        QTabBar::tab:selected {
          background: lightblue;
        }
        QTabBar::tab:hover {
           background: lightgray;
        }</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>菜肴</string>
    </attribute>
    <widget class="menu" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>151</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="menu" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>151</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="menu" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>151</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="menu" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>151</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="menu" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>151</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="menu" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>220</y>
       <width>151</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>主食</string>
    </attribute>
    <widget class="menu" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="menu" name="widget_8" native="true">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="menu" name="widget_9" native="true">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="menu" name="widget_10" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>饮料</string>
    </attribute>
    <widget class="menu" name="widget_11" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="menu" name="widget_12" native="true">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="menu" name="widget_13" native="true">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>110</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>menu</class>
   <extends>QWidget</extends>
   <header location="global">menu.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
